src/main/java/org/cp/confisual/nevisauth/AuthState.java

```diff
@@ -19,4 +21,9 @@
   public void addTransition(Transition transition) {
     transitions.add(transition);
   }
+
+  @Override
+  public void accept(NevisAuthVisitor nevisAuthVisitor) {
+    nevisAuthVisitor.visit(this);
```

Comment:
The logic to decide how to visit the authstate's transition should be implemented here.